### DISCUSSION TITLE
I did the impossible

### DISCUSSION THREAD

**Discussion by Pixel556** [DustinBrett/daedalOS#328] [Show and tell]
<img width="435" alt="Screenshot 2023-07-25 at 4 35 13 PM" src="https://github.com/DustinBrett/daedalOS/assets/95143545/2ed7c382-13e9-4cda-809e-a75dfde220c3">

**Comment by DustinBrett** [comment]
Haha cool, a nameless folder. I don't even recall if I disallowed this, I guess so. Is it using special characters or how did you do this?

My metric for if it's a bug or not is, does Windows 10 act the same way?

**Comment by DustinBrett** [comment]
I see now it can be done from the Terminal at least. I will stop this from happening at the Terminal. Did you do it from the UI? Based on the selection of the icon it seems like maybe it thinks there is some character there, taking up space. In my test case there was not even a character and the height reduced.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/c33958fa-9cdb-4bda-84cc-c6a1e3309a8c)

**Reply by Pixel556** [reply]
I used the infamous blank space character that is unrecognizable as a normal space, though some websites know this technique and they have blocked it.

**Reply by Pixel556** [reply]
Also, Windows acts the same way, though there is a legit method now for Windows 11 that is working to get a nameless folder. Though, the height won't reduce.

**Reply by DustinBrett** [reply]
Thanks for mentioning this. I think the one that works the same in Windows is ok. But the one I found using the Terminal allows actual spaces which is wrong, and also allows empty.

I will work on fixing the Terminal bug.
